news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,193 @@
+
+An example of a map which culminates this is their rendition of [Sta - 99 Glooms](https://osu.ppy.sh/beatmapsets/961775#mania/2013908). Its liberal usage of flams and more detailed interpretation of the track's various piano and synth are what make this map really shine. Another notable works of theirs is, [sasakure.UK - Jack-the-Ripper](https://osu.ppy.sh/beatmapsets/1130177#mania/2360897), a map featuring complex timing and even more complex representation of the track's guitar sounds. We hope to see Ticle's maps gain more traction as they are definitely worth a shot for any LN enthusiast!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+Get your editor ready. The mapping contests are here! Whether you are a veteran or making your first steps in mapping, contests are for everyone! In this section you can see what contests have been held in May or are ongoing!
+
+### Monthly Beatmapping Contests
+
+The [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest) (MBC) gives to you your monthly dose of mapping competitions. Each month's contest features a different mapping limitation that entries must adhere to.
+
+The April's contest results are out and we congratulate [hypercyte](https://osu.ppy.sh/users/9155377) for winning and getting his Tier 2 badge! Good job also to [fanzhen0019](https://osu.ppy.sh/users/418699) for being community vote runner-up and [Acyl](https://osu.ppy.sh/users/1943309) for being the screening crew favorite!
+
+The May and June Montly Beatmapping Contests have both their submission phase concluded and we can hope to see the results of it soon. Once they are out we make sure to let you know!
+
+### osu! Mapping Olympiad #7: osu!mania
+
+Keyboard mashers, rise up! It's your time to shine!
+The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode with every version. And this time only for the osu!mania mappers out there!
+
+Luring all mappers out there with Olympiad points for the top 3, which awards the Mapping Olympiad badge, and for the winner an unique user title, the "Elite Mapper"!
+
+Do you have what it takes to be the next "Elite Mapper"? Then jump right into it, grab one of the pre-timed .osz and start to map! We're excited to see all the mania maps!
+
+### 2020 osu!catch Mapping World Cup
+
+Fruit catcher, now's your chance! The 2020 edition of the [osu!catch Mapping World Cup](https://osu.ppy.sh/community/forums/topics/1089125) has arrived! If you are skilled with placing fruits in the editor get your 2 to 5 best friends by their hand and sign up! Can the Philippines keep their title as osu!catch Mapping World Champion or will an other country get the 1st place this time?
```

Comment:
If this newspost is out after 6/27 the description should remove mention of signing up because signups will be closed

Reply:
Changed the description and removed the sign up mention as they will be closed already with the release of this Mappers' report issue